go/processor/floria/calls_test.go

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package floria
+
+import (
+	"testing"
+
+	"github.com/Fantom-foundation/Tosca/go/tosca"
+	"go.uber.org/mock/gomock"
+)
+
+func TestCalls_InterpreterResultIsHandledCorrectly(t *testing.T) {
+	tests := map[string]struct {
```

Comment:
no test for error case ?

Reply:
There is a case for a failed interpreter execution in the test but it is only checking the success of the result. I think the output should not be considered in a failed case therefore I don't want to require a fixed value for it.